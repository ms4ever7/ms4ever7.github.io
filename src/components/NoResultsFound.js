import React from 'react';

const NoResultsFound = ({ selectedCategories, selectedColors }) => <div className='no-results-found__wrapper layout-column layout-align-center-center'>
  <h2 className='no-results-found__title'>No Results Found</h2>
  <h3 className='no-results-found__subtitle'>
    Sorry, but there are no products with such a colors and categories.
    </h3>
</div>


export default NoResultsFound;