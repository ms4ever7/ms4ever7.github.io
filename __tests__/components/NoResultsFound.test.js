import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import NoResultsFound from '../../src/components/NoResultsFound';

describe('NoResultsFound', () => {
  let wrapper;


  it('should render correctly', () => {
    wrapper = shallow(
      <NoResultsFound />
    );

    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });

  it('has appropriate classnames', () => {
    const wrapper = shallow(<NoResultsFound />);

    expect(wrapper.find('.no-results-found__wrapper').exists()).toBeTruthy();
    expect(wrapper.find('.no-results-found__title').exists()).toBeTruthy();
    expect(wrapper.find('.no-results-found__subtitle').exists()).toBeTruthy();
  });
});
