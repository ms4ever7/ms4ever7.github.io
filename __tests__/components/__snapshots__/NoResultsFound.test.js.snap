// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResultsFound should render correctly 1`] = `
<div
  className="no-results-found__wrapper layout-column layout-align-center-center"
>
  <h2
    className="no-results-found__title"
  >
    No Results Found
  </h2>
  <h3
    className="no-results-found__subtitle"
  >
    Sorry, but there are no products with such a colors and categories.
  </h3>
</div>
`;
